src/Notes/WC_Admin_Notes.php

```diff
@@ -100,6 +103,74 @@ public static function delete_notes_with_name( $name ) {
 		}
 	}
 
+	/**
+	 * Update a note.
+	 *
+	 * @param WC_Admin_Note $note The note that will be deleted.
```

Comment:
This should be "The note that will be updated"

Reply:
Totally! Nice catch Timmy. This was modified in the commit [d10a504](https://github.com/woocommerce/woocommerce-admin/pull/4262/commits/d10a5043c34c1278387376bd467f9e1df727dcb4)